src/booleans.js

```diff
@@ -1,59 +1,123 @@
 export const negate = (a) => {
-  // your code here
+  return a === false;
 };
 
 export const both = (a, b) => {
-  // your code here
+  if (a && b === true){
```

Comment:
Due to operator precedence, this equation only checks that `a` is truthy, and that `b` is strictly `true` (because `&&` takes precedence over `===`) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence